apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -228,6 +229,8 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
     @ViewChild('advanceReceiptComponent', {static: true}) public advanceReceiptComponent: AdvanceReceiptAdjustmentComponent;
 
     @ViewChild('dateChangeConfirmationModel', {static: true}) public dateChangeConfirmationModel: ModalDirective;
+    /** amount filed component */
+    @ViewChild('AmountFieldComponent', { static: true }) public amountField: ElementRef;
```

Comment:
Please check if it is required as it is unused

Reply:
Removed